src/components/detail/DetailHeader.tsx

```diff
@@ -0,0 +1,97 @@
+import { Text } from "@/components/ui/text";
+import { OverviewStat } from "@/components/companies/detail/overview/OverviewStat";
+import { SectionWithHelp } from "@/data-guide/SectionWithHelp";
+import { DataGuideItemId } from "@/data-guide/items";
+import { useTranslation } from "react-i18next";
+import { PoliticalRuleLabel } from "./PoliticalRuleLabel";
+import { ReactNode } from "react";
+
+export interface DetailStat {
+  label: string | ReactNode;
+  value: string | ReactNode;
+  unit?: string;
+  valueClassName?: string;
+  info?: boolean;
+  infoText?: string;
+}
+
+export interface DetailHeaderProps {
+  name: string;
+  subtitle?: string;
+  logoUrl?: string | null;
+  politicalRule?: string[];
+  helpItems: DataGuideItemId[];
+  stats: DetailStat[];
+  translateNamespace: string;
+  politicalRuleLabelKey?: string;
+}
+
+export function DetailHeader({
+  name,
+  subtitle,
+  logoUrl,
+  politicalRule,
+  helpItems,
+  stats,
+  translateNamespace,
+  politicalRuleLabelKey = "politicalRule",
+}: DetailHeaderProps) {
+  const { t } = useTranslation();
+
+  const politicalRuleLabels = politicalRule?.map((p, index) => (
```

Comment:
Do we want this in a generalised component since it's pretty tied to local data?

Reply:
Yeah that sounds good 🙂